octobot_trading/traders/trader.py

```diff
@@ -232,12 +232,16 @@ async def sell_all(self, currencies_to_sell=None, timeout=None):
                 orders += await self._sell_everything(symbol, inverted, timeout=timeout)
         return orders
 
-    async def notify_order_cancel(self, order):
+    async def notify_order_cancel(self, order, remove_from_manager=False):
```

Comment:
Why this is False by default ?

Reply:
Most of the time, the order is removed in ```notify_order_close``` except when it's a self-managed order, therefore the most common case is not removing it from there.

I think we should refactor this order closing/cancelling part, it's a very complicated process from 0.3 that we could simplify

Reply:
Ok :+1: